plasma_framework/truffle-config.js

```diff
@@ -92,15 +92,14 @@ module.exports = {
     // Configure your compilers
     compilers: {
         solc: {
-            // version: "0.5.1",    // Fetch exact version from solc-bin (default: truffle's version)
-            // docker: true,        // Use "0.5.1" you've installed locally with docker (default: false)
-            // settings: {          // See the solidity docs for advice about optimization and evmVersion
-            //  optimizer: {
-            //    enabled: false,
-            //    runs: 200
-            //  },
-            //  evmVersion: "byzantium"
-            // }
+            version: '0.5.11',
+            settings: {
+                optimizer: {
+                    enabled: true,
+                    runs: 200,
+                },
+                evmVersion: 'byzantium',
```

Comment:
Why 'byzantium'? The latest is 'petersburg'  (which is the default).

Reply:
ah....I just uncomment it and thought that's the latest @@. I think I would just remove then, unless there's reason to explicitly state that.